frontend/src/features/map_planning/layers/plant/components/PlantListItem.tsx

```diff
@@ -34,6 +34,8 @@ export function PlantListItem({ plant, onClick, isHighlighted = false }: PlantLi
           defaultImageUrl={defaultImageUrl}
           path={`Icons/${plant?.unique_name}.png`}
           shareToken="2arzyJZYj2oNnHX"
+          retry={false}
```

Comment:
Did you try something like this? Maybe it makes sense to retry in the other cases.
Code does not work here, but such a function can be passed to the `useQuery` hook.
```ts
          retry={(failureCount, error) => {
              if(check if error is network 404 error) return false;
              return true;
          }}
```